frontend/public/config.yml

```
@@ -443,12 +443,19 @@
       - { label: 'Nadpis', name: 'title', widget: 'string' }
       - { label: 'Dátum', name: 'date', widget: 'string' }
       - { label: 'Miesto', name: 'place', widget: 'string' }
-      - { label: 'Instagram', name: 'instagram', widget: 'string' }
       - { label: 'Facebook Event', name: 'linkFbEvent', widget: 'string' }
-      - { label: 'Discord', name: 'discord', widget: 'string' }
       - { label: 'Obrázok', name: 'image', widget: 'image' }
       - { label: 'Podnadpis', name: 'subTitle', widget: 'string' }
       - { label: 'Anotácia', name: 'annotation', widget: 'string' }
+      - { label: 'Veľký text nad zápletkami', name: 'storyIntroText', widget: 'string', default: 'Na čo sa môžeš tešiť?' }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)